src/shout_subgroup/utils.py

```diff
@@ -29,35 +31,28 @@ async def is_group_chat(telegram_chat_id: int) -> bool:
     return telegram_chat_id <= 0
 
 
-async def format_telegram_usernames(usernames: set[str], telegram_user: User) -> set[str]:
+async def replace_me_mentions(usernames: set[str], telegram_user: User) -> set[str]:
```

Comment:
B/c we're using `user_id` now. We don't need to format usernames. But we still need to replace @me with the username